File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.java
import java.util.Scanner;
import java.util.Arrays;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        
        int[] number = new int[num];
        
        for(int i=0; i<num; i++){
            number[i] = sc.nextInt();
        }
        
        Arrays.sort(number);
        
        for(int i=0; i < num; i++){
            System.out.println(number[i]);
        }
    }
}

File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.kt
import java.util.Scanner;

fun main(){
    val sc = Scanner(System.`in`)
    val num = sc.nextInt()
    val number = IntArray(num)
        
    for(i in 0 until num){
        number[i] = sc.nextInt()
    }
        
    number.sort()
        
    for(i in 0 until num){
        println(number[i])
    }
}